### PR TITLE
Update the Articles 

### PR DESCRIPTION
**Pull request by Dilnu**
Also make a first pass at updating the easy version of the articles to match.

### CODE REVIEW

**Comment by Dilnu** [comment]
Not sure if we want to change the formatting from the current style to something else. I'm also not sure how to preview the changes so maybe there's mistakes in the formatting of the file I exported from dropbox.

**Comment by jonahbron** [comment]
Would you provide some context to this PR?  What's the reason for having a new articles page in addition to the existing one?

**Comment by Dilnu** [comment]
I should probably update to delete the PDF version. The goal is to have them be easier to read on mobile by not being a PDF.

**Comment by Dilnu** [comment]
Also to be clear this is updating based on the proposals passed by the recent assembly

File: frontend/content/principles/articles-full.en.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
+---
+title: 'Full Articles'
```
Comment (realjenny) [inline frontend/content/principles/articles-full.en.md:2]: Can you change the title to be `Full articles`. The website uses sentence case style for titles.

**Review by realjenny — APPROVED** [review]

**Pull request merged**